framework/auth/views.py

```diff
@@ -137,8 +140,11 @@ def auth_login(auth, **kwargs):
         # Only allow redirects which are relative root or full domain, disallows external redirects.
         if not (next_url[0] == '/' or next_url.startsWith(settings.DOMAIN)):
             raise HTTPError(http.InvalidURL)
-
     if auth.logged_in:
+        # remove expired email verifications
+        if not campaign:
+            auth.user.clean_email_verifications(auth.user)
```

Comment:
I am getting a TypeError here because `clean_email_verifications` does not expect arguments.

```
Traceback (most recent call last):
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/sloria/projects/cos-projects/osf/framework/routing/__init__.py", line 118, in wrapped
    data = fn(*args, **kwargs)
  File "/Users/sloria/projects/cos-projects/osf/framework/auth/decorators.py", line 22, in wrapped
    return func(*args, **kwargs)
  File "/Users/sloria/projects/cos-projects/osf/framework/auth/views.py", line 146, in auth_login
    auth.user.clean_email_verifications(auth.user)
TypeError: clean_email_verifications() takes exactly 1 argument (2 given)
```